src/pages/Search/components/SearchCard/SearchCard.module.css

```diff
@@ -1,4 +1,71 @@
 .goBtn {
+
+    margin: 10px 0;
+    padding: 10px 20px;
+    background-color: var(--primary);
+    color: var(--white);
+    font-weight: bold;
+    border-color: var(--primary);
+ }
+ .btnFooter{
+  
+   padding: 10px 20px;
+   background-color: var(--primary);
+   color: var(--white);
+   font-weight: bold;
+   border-color: var(--primary);
+   width:100%;
+  
+ }
+ .footer{
+   background-color: var(--primary);
+   display:none;
+ }
+ 
+ .goBtn:hover {
+    background-color: var(--dark-primary);
+ }
+ .btnFooter:hover {
+   background-color: var(--dark-primary);
+   border: 1px solid var(--dark-primary);
+}
+.footer:hover{
+   background-color: var(--dark-primary);
+}
+ .cardImg {
+    display: inline-block;
+    object-fit: cover;
+ }
+ .card {
+   width: calc(50% - 1rem); 
+   margin-bottom: 1rem; 
+ }
+ .background{
+   position: relative;
+   background-image: url('src/assets/images/sample-pic-horizontal.jpg');
+ }
+ @media screen and (max-width: 840px){
+   .goBtn{
+      display:none;
+   }
+   .footer{
+      display:block;
+    }
+    .description{
+      display: none;
+    }
+    
+ }
+ @media screen and (max-width: 575px){
+   .card {
+      width: 100%;
+      margin-bottom: 1rem; 
+    }
+    .footer{
+      display:block;
+    }
+ }
+=======
```

Comment:
Please run your local file and solve all the errors before requesting others to review

Reply:
i updated this one and now it is working on my local file as well.